### PR TITLE
podman: experiment with "fallback to normal bootstrap"

### PR DESCRIPTION
**Pull request by praiskup**
Needs to be applied on top of #1199, if at all.

### CODE REVIEW

**Review by xsuchy — APPROVED** [review]
+1

**Comment by praiskup** [comment]
@xsuchy can you please re-review?  I've also added retry-mechanism.

File: mock/py/mockbuild/buildroot.py [hunk 1]
```diff
@@ -211,6 +212,33 @@ def _init_locked(self):
...
 
+    @traceLog()
+    def _try_to_load_from_container_image(self):
+
+        @contextmanager
+        def _fallback(message):
+            try:
+                yield
+            except BootstrapError as exc:
+                if not self.config["image_fallback"]:
```
Comment (xsuchy) [inline mock/py/mockbuild/buildroot.py:223]: image_fallback or bootstrap_image_fallback?

Comment (xsuchy) [reply mock/py/mockbuild/buildroot.py:223]: Ahh, in bootstrap it is without the prefix. Ignore my comment.

**Review by xsuchy — APPROVED** [review]
+1

**Comment by praiskup** [comment]
Thank you for taking a look!

**Comment by praiskup** [comment]
rebased

**Comment by praiskup** [comment]
Sorry for yet another rebase, I believe the code is now a bit easier.

**Pull request merged**